lib/graphql_devise/route_mounter.rb

```diff
@@ -1,6 +1,7 @@
 module GraphqlDevise
   module RouteMounter
     def mount_graphql_devise_for(resource, options = {})
+      GraphqlDevise.setup_base_controller(options.delete(:base_controller))
```

Comment:
What if we build a new controller class every time the method is called? Maybe something like:

```ruby
# do not execute if base_controller option is not present
new_controller = GraphqlDevise.const_set("#{resource}AuthController", Class.new(options.delete(:base_controller)))

new_controller.include(SetUserByToken)
new_controller.include(AuthActionMethod) # module that defines the controller action

post clean_options.at, to: "#{new_controller.to_s.underscore}#auth"
get clean_options.at, to: "#{new_controller.to_s.underscore}#auth"
```

Not sure that works, but it should?